docs/customservices.md

```diff
@@ -141,3 +140,20 @@ See https://docs.portainer.io/v/ce-2.11/user/account-settings#access-tokens
   type: "Portainer"
   apikey: "MY-SUPER-SECRET-API-KEY"
 ```
+
+## Uptime Kuma
+
+Using the Uptime Kuma service you can display info about your instance uptime right on your Homer dashboard.
+
+The following configuration is available for the UptimeKuma service. Needs v1.13.1 or later because of the change in APIs due to [multiple status pages support](https://github.com/louislam/uptime-kuma/releases/tag/1.13.1).
```

Comment:
Does the whole custom service require the 1.13.1 API or just the slug/multi status page support?

If the whole thing requires it, I'd recommend rephrasing this line to just simply be "Requires Uptime Kuma version 1.13.1 or later." and scrapping the mention of the config or the link out to their repo.

If only the slug feature requires it, clarify as such here and as a comment in the example yaml file

Reply:
The whole service needs it. 1.13.1 changed the API structure along with introducing multiple status pages. So the component can support either the old API format or the new one, not both. So went with the new format.